service/src/test/java/org/mskcc/smile/service/CorrectCmoPatientIdHandlerTest.java

```diff
@@ -41,7 +41,7 @@ public class CorrectCmoPatientIdHandlerTest {
     private SmilePatientService patientService;
 
     @Container
-    private static final Neo4jContainer databaseServer = new Neo4jContainer<>("neo4j:4.4")
+    private static final Neo4jContainer<?> databaseServer = new Neo4jContainer<>()
```

Comment:
Note that I added the wildcard `<?>` to parameterize the raw type `Neo4jContainer`. Did this following the example from the [latest docs](https://java.testcontainers.org/modules/databases/neo4j/) for best practices and because the new `TestContainer` version is stricter about this (I got some errors when running `mvn clean install` without adding this wildcard parameter).